Features/Context/SubContext/CommonActions.php

```diff
@@ -231,17 +231,45 @@ public function clickEditActionBar($button)
     }
 
     /**
-     * @Given I click (on) the content tree with path :path
-     * @Then I see :path in the content tree
-     * Explores the content tree, expanding it and click on the desired element
+     * @Then There is not a content :path
+     * @Then There isn't a content :path
+     * Explores the finder of the UDW, verify the desired element doesn't exist and close the UDW.
      *
-     * @param   string  $path    The content tree path such as 'Content1/Content2/ContentIWantToClick'
+     * @param   string  $path    The content browse path such as 'Content1/Content2/ContentIWantToClick'
      */
-    public function clickOnTreePath($path)
+    public function verifyContentIsMissing($path)
     {
-        $node = $this->findWithWait('.ez-view-discoverybarview');
-        $this->clickDiscoveryBar('Content tree');
-        $this->openTreePath($path, $node);
+        $this->clickNavigationItem('Content structure');
+        $this->dontSeeBrowsePath($path);
+        $this->cancelSelection();
+    }
+
+    /**
+     * @Then There is a content :path
+     * Explores the finder of the UDW, find the desired element and close the UDW.
+     *
+     * @param   string  $path    The content browse path such as 'Content1/Content2/ContentIWantToClick'
```

Comment:
No need to add the extra spaces, we don't align statements. And there is nothing to align with :)